Source/VMware.vSphereDSC/DSCResources/VMHostNetwork/VMHostVss/VMHostVss.ps1

```diff
@@ -188,10 +160,8 @@ class VMHostVss : VMHostBaseDSC {
             $vssConfigArgs.Add('Operation', 'remove')
         }
 
-        $vssConfig = New-VssConfig @vssConfigArgs
-        $vmHostNetworkSystem = Get-View -Server $this.Connection -Id $vmHost.ExtensionData.ConfigManager.NetworkSystem
         try {
-            Update-Network -NetworkSystem $vmHostNetworkSystem -Type 'VSS' -VssConfig $vssConfig
+            Update-Network -NetworkSystem $this.vmHostNetworkSystem -VssConfig $vssConfigArgs
         }
         catch {
             Write-Error "The virtual switch Config could not be updated: $($_.Exception.Message)"
```

Comment:
Change to "The Virtual Switch Config could not be updated: $($_.Exception.Message)."

Reply:
See previous comment